lib/rules/selector-max-universal/README.md

```diff
@@ -74,3 +74,35 @@ The following patterns are _not_ considered problems:
 /* `*` is inside `:not()`, so it is evaluated separately */
 * > * .foo:not(*) {}
 ```
+
+## Optional secondary options
+
+### `ignoreAfterCombinators: [">", "+", "~"]`
+
+Ignore universal selector if it's preceded by specified after combinators.
+
+Given:
+
+```json
+[1] // maximum universal selectors allowed
+[">", "+"]
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+*.foo * {}
+```
+
+While the following patters are _not_ considered problems:
```

Comment:
[suggest] Please keep consistency:

```suggestion
The following patters are _not_ considered problems:
```

Because the following script in our website repo would not work...

https://github.com/stylelint/stylelint.io/blob/4e562026b6c5aa0aa61ad0c437dba8881a936e08/scripts/generate-docs.mjs#L62-L63

Reply:
We should replace it [here](https://github.com/stylelint/stylelint/blob/main/lib/rules/selector-max-id/README.md) too.
![image](https://user-images.githubusercontent.com/34133492/185091282-106fd657-a13f-411b-956c-3c86a9524e63.png)

Reply:
Ooops. The following area should be green... 😓 

<img width="477" alt="image" src="https://user-images.githubusercontent.com/473530/185092390-4be3146e-226f-4af8-a507-c3eef86719df.png">

Could you please open a PR to fix the doc?

https://github.com/stylelint/stylelint/blob/2c2a57cdca138aa6afe225b90b3fa6d2110245fc/lib/rules/selector-max-id/README.md?plain=1#L97

Reply:
> Could you please open a PR to fix the doc?

Will be glad to help!

Reply:
Here we go #6276